specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -189,6 +198,31 @@ export function FormTable<SCHEMA extends AnySchema>({
     scrollerRef
   );
 
+  const blockers = useAllSaveBlockers(parentResource, relationship);
```

Comment:
This code shouldn't be here as you already have it in IntegratedRecordSelector, which covers both form table and record selector